### PR TITLE
Add support for easier monitoring

### PR DESCRIPTION
**Pull request by KittyLe**
This PR adds names to each mirror, making it easier to understand whose and which one has failed in monitoring.

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 pkgname=cyberos-mirrorlist
 pkgver=20210510
-pkgrel=4
+pkgrel=5
```
Comment (omaemae) [inline PKGBUILD:5]: you updated the pkgrel instead of the date!

**Review by omaemae — CHANGES_REQUESTED** [review]

File: PKGBUILD (hunk 1 above)
Comment (omaemae) [reply PKGBUILD:5]: RTFM https://github.com/cyberos/docs/blob/main/source/developer/mirror.md#becoming-a-tier-2-mirror

Comment (KittyLe) [reply PKGBUILD:5]: this actually might be a case where 2 people think differently: pkgrel is pkg release, right? you might think that it is just when mirrors update, I thought that it is for every change that actually changes stuff.

Comment (KittyLe) [reply PKGBUILD:5]: > RTFM https://github.com/cyberos/docs/blob/main/source/developer/mirror.md#becoming-a-tier-2-mirror
meh, give me a sec

**Pull request merged**